plugins/woocommerce/src/Blocks/BlockTypes/ProductCollection.php

```diff
@@ -324,6 +434,7 @@ public function add_support_for_filter_blocks( $pre_render, $parsed_block ) {
 	 * @return array
 	 */
 	public function build_frontend_query( $query, $block, $page ) {
+
```

Comment:
```suggestion
```

Seems like unnecessary empty line

Reply:
Thanks so much for the review, @kmanijak! I took care of these issues!